dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -80,7 +80,8 @@
  * @author dyuen
  */
 @Entity
-@ApiModel(value = "Version", description = "Base class for versions of entries in the Dockstore")
+@ApiModel(value = "Version", description = Version.VERSION_DESCRIPTION)
+@Schema(name = "Version", description = Version.VERSION_DESCRIPTION, subTypes = {WorkflowVersion.class, Tag.class})
```

Comment:
I guess this doesn't need a discriminator like Workflow does, because a WorkflowVersion is the type when it's Workflow, and Tag is the type when it's a Container, so there should be no deserialization problems? There must not be with the passing tests...

Reply:
I believe so